Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -543,20 +538,15 @@ final class AttributeTests: XCTestCase {
 
     assertParse(
       """
-      @_unavailableFromAsync(message: 1️⃣abc2️⃣)
+      @_unavailableFromAsync(message: 1️⃣abc)
       func foo() {}
       """,
       diagnostics: [
         DiagnosticSpec(
           locationMarker: "1️⃣",
-          message: #"expected '"' in string literal"#,
-          fixIts: [#"insert '"'"#]
-        ),
-        DiagnosticSpec(
-          locationMarker: "2️⃣",
-          message: #"expected '"' to end string literal"#,
-          fixIts: [#"insert '"'"#]
-        ),
+          message: #"expected 'abc' to be surrounded by '"'"#,
+          fixIts: [#"insert '""'"#]
```

Comment:
I suspect that this will change to `insert '"' and '"'` after https://github.com/apple/swift-syntax/pull/1651 is merged.

Reply:
Thank you for letting me know! Once the PR is merged, I will rebase to apply the changes 🙂